bliss/metrics.py

```diff
@@ -40,50 +43,60 @@ def __init__(
             fp: false positives = # of predicted sources not matched with true source
             avg_distance: Average l-infinity distance over matched objects.
             total_true_n_sources: Total number of true sources over batches seen.
-            total_correct_class: Total # of correct classifications over matched objects.
-            total_n_matches: Total # of matches over batches.
-            conf_matrix: Confusion matrix (galaxy vs star)
             disable_bar: Whether to show progress bar
+            gal_tp: true positives = # of sources correctly classified as a galaxy
```

Comment:
Could we rename `tp` and `fp` something like `detection_tp` and `detection_fp` to better distinguish them from `gal_tp` and `gal_fp`?

Reply:
Will do! It's definitely an unclear naming convention as-is.